releasenotes/README.md

```diff
@@ -19,7 +19,14 @@ These changes are in the Salesforce CLI release candidate. We plan to include th
 * NEW: Get verbose CLI version information in JSON format with just a single command. Use the new `--json` and `--verbose` parameters of `sfdx version`. It's now a smidgen easier to gather your environment details for us when you report an issue:
 
 	`sfdx version --json --verbose`
```

Comment:
If you want --verbose output, you must also specify --json? Can you run these options separately? Or is now that you can get verbose JSON output? When I ran the command alone, with --json, and with --verbose, I got the exact same results/output.

Reply:
Hmmm, i see very diff output with the various param combos.  Are you sure you're on the RC (7.108)?  I'll reword to make it clear you can use 'em separately.

Reply:
Just for kicks, I used the current CLI but wanted to see if these options did anything.